QUEST_LV_0200.tsv

```diff
@@ -9063,27 +9063,27 @@ QUEST_LV_0200_20150908_009066	You've been pricked by the thorns of bushes!
 QUEST_LV_0200_20150908_009067	Digging up the gold plate
 QUEST_LV_0200_20150908_009068	You fell down while you were digging up the gold plate, but you've digged it well!
 QUEST_LV_0200_20150908_009069	Monsters suddenly appeared near the gold plate!
-QUEST_LV_0200_20150908_009070	You've digged the gold plate well!
-QUEST_LV_0200_20150918_009071	
-QUEST_LV_0200_20150918_009072	
-QUEST_LV_0200_20150918_009073	
-QUEST_LV_0200_20150918_009074	
-QUEST_LV_0200_20150918_009075	
-QUEST_LV_0200_20150918_009076	
-QUEST_LV_0200_20150918_009077	
-QUEST_LV_0200_20150918_009078	
-QUEST_LV_0200_20150918_009079	
-QUEST_LV_0200_20150918_009080	
-QUEST_LV_0200_20150918_009081	
-QUEST_LV_0200_20150918_009082	
-QUEST_LV_0200_20150918_009083	
-QUEST_LV_0200_20150918_009084	
-QUEST_LV_0200_20150918_009085	
-QUEST_LV_0200_20150918_009086	
-QUEST_LV_0200_20150918_009087	
-QUEST_LV_0200_20150918_009088	
-QUEST_LV_0200_20150918_009089	
-QUEST_LV_0200_20150918_009090	
+QUEST_LV_0200_20150908_009070	You've dug the gold plate well!
+QUEST_LV_0200_20150918_009071	$I've been able to regain some of my power thanks to you, but this is only the beginning  -- {nl} I'll help protect you now by luring the demons with the fake revelation at the tower. {nl}
+QUEST_LV_0200_20150918_009072	$Please take Laima's revelation to the Cathedral. Believe in yourself and never hold any doubts. Victory is certainly yours in the end.
```

Comment:
"Victory is certainly yours in the end." -> "Victory will certainly be yours in the end."

?